presto-hive/src/test/java/com/facebook/presto/hive/TestMergeJoinPlan.java

```diff
@@ -83,19 +90,19 @@ public void testJoinType()
             assertPlan(
                     mergeJoinEnabled(),
                     "select * from test_join_customer_join_type left join test_join_order_join_type on test_join_customer_join_type.custkey = test_join_order_join_type.custkey",
-                    joinPlan("test_join_customer_join_type", "test_join_order_join_type", ImmutableList.of("custkey"), ImmutableList.of("custkey"), LEFT, false));
+                    joinPlan("test_join_customer_join_type", "test_join_order_join_type", ImmutableList.of("custkey"), ImmutableList.of("custkey"), LEFT, true));
```

Comment:
Now not only inner joins are supported

Reply:
Only INNER join is supported -> is this true for Prestissimo only?